src/_lib/modules/dropdowns/dropdown-toggle.directive.ts

```diff
@@ -50,6 +52,10 @@ export class DropdownToggleDirective implements OnInit, OnDestroy {
         }
     }
 
+    private _isMobile() {
```

Comment:
I'd suggest renaming to _isMobileScreenSize(), because a small screen is not an indicator of a mobile device. (I do realise actual device detection is notoriously difficult - as someone once described it, "it's a bag of lies".)